Examples/NavigationToolbarExample.qml

```diff
@@ -17,27 +17,35 @@
 import QtQuick 2.0
 import QtQuick.Controls 1.1
 import QtPositioning 5.2
-import Esri.ArcGISRuntime 100.0
+import Esri.ArcGISRuntime 100.00
 import Esri.ArcGISRuntime.Toolkit.Controls 1.1
 
 Rectangle {
     width: 500
     height: 400
 
-    Map {
-        id: map
+    MapView {
+        id: mapview
         anchors.fill: parent
-        wrapAroundEnabled: true
-        mapPanningByMagnifierEnabled: true
-        magnifierOnPressAndHoldEnabled: true
+        wrapAroundMode: Enums.WrapAroundModeDisabled
+        magnifierEnabled: true
         zoomByPinchingEnabled: true
-        positionDisplay {
-            positionSource: PositionSource {
+        allowMagnifierToPanMap: true
+        rotationByPinchingEnabled: true
+
+        Map {
+            Basemap {
```

Comment:
Might be cleaner to use BasemapNationalGeographic {} since it is just using the Esri basemap